### PR TITLE
Add suggestion on how to use demo

### PR DESCRIPTION
**Pull request by pgiraud**
With small modifications on 'demo' block layout

screenshot from 2017-07-11 14-57-24

### CODE REVIEW

File: _includes/demo.html [hunk 1]
```diff
@@ -10,13 +10,21 @@
...
+      <div class="col-lg-4 col-lg-offset-4 text-center">
+        <a href="http://demo-powa.dalibo.com" class="btn btn-lg btn-outline contrast">
+          <i class="fa fa-share"></i> Try it now
+        </a>
+        <br>
+        <p>
+        <span class="help-block">
+        login : powa<br>pass : demo
+        </span>
+        <span class="help-block">Tip: Check the <em><strong>tcp</strong></em> database if you want to see how the <em>index suggestion</em> works.</span>
```
Comment (rjuju) [inline _includes/demo.html]: Small typo, this is the **tpc** database, not **tcp** :)

File: css/bootstrap.min.css [hunk 2]
```diff
@@ -4,4 +4,4 @@
...
...nline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn,.navbar-form .input-group .form-control{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .radio,.navbar-form .checkbox{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .radio label,.navbar-form .checkbox label{padding-left:0}.navbar-form .radio input[type="radio"],.navbar-form .checkbox input[type="checkbox"]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}}@media (min-width:768px){.navbar-form{width:auto;border:0;margin-left:0;margin-right:0;padding-top:0;padding-bottom:0;-webkit-box-shadow:none;box-shadow:none}.navbar-form.navbar-right:last-child{margin-right:-15px}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-right-radius:0;border-top-left-radius:0}.navba
```
Comment (rjuju) [inline css/bootstrap.min.css]: I'm not against updating boostrap, but it looks like the version is the same.  I didn't try to see where the change were, do you have any info about this?

**Comment by rjuju** [comment]
I just reviewed the PR.  Except the small issue, this is a really great improvement. Nice job!

File: _includes/demo.html (hunk 1 above)
Comment (pgiraud) [inline _includes/demo.html]: Fixed. Thanks.

File: css/bootstrap.min.css (hunk 2 above)
Comment (pgiraud) [inline css/bootstrap.min.css]: Crap. I have just seen your comment. This file doesn't seem to be the original one. It's a customized version of bootstrap with specific colors.
The change I made is for the color on the "help-block" rule.

**Pull request merged**